public/app/features/logs/components/LogDetails.tsx

```diff
@@ -71,7 +71,8 @@ class UnThemedLogDetails extends PureComponent<Props> {
     const labelsAvailable = Object.keys(labels).length > 0;
     const fieldsAndLinks = getAllFields(row, getFieldLinks);
     const links = fieldsAndLinks.filter((f) => f.links?.length).sort();
-    const fields = fieldsAndLinks.filter((f) => f.links?.length === 0).sort();
+    // do not show the log message unless there is a link attached
+    const fields = fieldsAndLinks.filter((f) => f.links?.length === 0 && f.fieldIndex !== row.entryFieldIndex).sort();
```

Comment:
@ifrost because LogParser doesnt know what links are not filtered out (the filtering we added to remove links that aren't interpolated happens after the parsing), I change the logic to filter out that here.